docs/google-kubernetes-tutorial.md

```diff
@@ -79,7 +79,6 @@ Copy `helm` to your `$HOME` directory so that it will persist after the Cloud Sh
 
 Helm will also need a couple of permissions to work properly:
 
-	kubectl create serviceaccount tiller --namespace kube-system &&
 	kubectl apply -f ./manifests/tiller-rbac.yaml &&
```

Comment:
We should remind users that they should enter `tidb-operator` workspace first when execute this command (based on tangliu's experience today).

Reply:
By default, the user enters into cloned tidb-operator directory automatically. I guess he changed working directory during testing.

One way to avoid this is to use absolute path `$HOME/tidb-operator/...` in commands, but it does not always apply because when users enter console again and they may clone tidb-operator.git into another directory (e.g. tidb-operator-2).

Reply:
No, the GCP tutorial will prompt users to run `git pull` instead of `git clone` again.

Reply:
Yes, but user may choose the third choice. 

```
fuyecheng@cloudshell:~$ cloudshell_open --repo_url "https://github.com/cofyc/tidb-operator" --page "editor" --tutorial "docs/google-kubernetes-tutorial.md" --git_branch "gke-tutorial"
  You have already cloned this repo into directory tidb-operator. Would you like to:
  [1] cd into that directory
  [2] cd into that directory and `git pull`
  [3] git clone a new copy
  Enter your choice [default 1]:
```

Reply:
Okay, it is hard to cover all use cases. The `$HOME/tidb-operator` approach lgtm.

Reply:
I reverted `$HOME/tidb-operator` because of this (directory name may change). I've tried the tutorial several times, using the relative path is better because cloud shell always cd into the cloned directory.